### PR TITLE
docs: minikube uses Alibaba Cloud image , vmware driver and docker driver

### PR DESCRIPTION
**Pull request by KeHaohaoke**
# Summary

Minikube uses Alibaba Cloud image

## Description

Minikube uses Alibaba Cloud image, two methods: Vmware driver ,  docker driver

## screenshot

**from my own site**

image

### CODE REVIEW

File: content/posts/使用minikube和阿里云镜像安装k8s/index.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+author: "HaoKe | 柯浩"
+authorLink: http://github.com/KeHaohaoke
+tags: ["kubernetes", "minikube"]
+categories: ["kubernetes","minikube","General","中文文章"]
+date: 2022-05-18
+---
+
+我的同事胡涛写了一篇英文的博客，关于如何使用minikube搭建本地的k8s环境的。
+
+我很喜欢这篇文章,链接在这里：[minikube-k8s](../getting-started-with-minikube)。
```
Comment (daniel-hutao) [inline content/posts/使用minikube和阿里云镜像安装k8s/index.md]: , -> ，

File: content/posts/使用minikube和阿里云镜像安装k8s/index.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+安装也很简单：
+
+遵循这里的guide去做就OK了。[macos-docker](https://docs.docker.com/desktop/mac/install/)
+
+作者在这里也去说明一下安装方法:
+
+1. 去docker官网的链接，找到你的机器对应的安装，点击下载。
+[docker-desktop-install](https://docs.docker.com/desktop/mac/install/)
+
+
```
Comment (daniel-hutao) [inline content/posts/使用minikube和阿里云镜像安装k8s/index.md]: duplicated blank line

File: content/posts/使用minikube和阿里云镜像安装k8s/index.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+![install-docker-desktop](./install-docker-desktop.png)
+
+>如果有喜欢命令行安装的朋友，可以看这个链接 [install-docker-desktop-from-command-line](https://docs.docker.com/desktop/mac/install/#install-from-the-command-line)
+
+安装完成之后，就可以启动docker desktop。
+
+> Tips: 如果Mac配置不高，比如作者的Mac，买的比较早一些，可以在启动docker desktop之前，设置docker desktop里的内存小一点。
+
+修改docker desktop的内存的步骤： 在MAC右上角的状态栏找到docker desktop->Preferences->按需修改内存和CPU的设置。
```
Comment (daniel-hutao) [inline content/posts/使用minikube和阿里云镜像安装k8s/index.md]: "： " -> "："

File: content/posts/使用minikube和阿里云镜像安装k8s/index.md (hunk 1 above)
Comment (KeHaohaoke) [reply content/posts/使用minikube和阿里云镜像安装k8s/index.md]: Has been repaired.

File: content/posts/使用minikube和阿里云镜像安装k8s/index.md (hunk 2 above)
Comment (KeHaohaoke) [reply content/posts/使用minikube和阿里云镜像安装k8s/index.md]: Has been repaired.

File: content/posts/使用minikube和阿里云镜像安装k8s/index.md (hunk 3 above)
Comment (KeHaohaoke) [reply content/posts/使用minikube和阿里云镜像安装k8s/index.md]: Has been repaired.

**Review by daniel-hutao — APPROVED** [review]

**Pull request merged**